jira-automation.py

```diff
@@ -0,0 +1,284 @@
+import json
+import logging
+import os
+import sys
+import argparse
+
+from utils.utils import (
+    exclude_file,
+    format_code_analysis_results,
+    get_code_analysis_results,
+    parse_file_name,
+    parse_project_name,
+)
+from models.models import JiraClient, SnykClient, VulnerabilityData
```

Comment:
isn't better import the .py file directly instead importing each method?

Reply:
yes it is :). Done that